ulii/urls.py

```diff
@@ -1,8 +1,15 @@
-from django.urls import include, path
+from django.urls import include, path, re_path
 
-from senlii import views
+from liiweb.views import RedirectCaseURLsView
+from ulii import views
 
 urlpatterns = [
     path("", views.HomePageView.as_view(), name="home_page"),
+    # This redirects old Ulii case laws to work with peachjam urls
+    re_path(
+        r"^ug/judgment/(?P<court>[-\w]+)/(?P<year>\d+)/(?P<number>\d+)$",
```

Comment:
I suggest making the `ug` a regex component `[a-z]{2}` and then all of this can move into the liiweb base, and just be generic.

Reply:
Done